src/shared/templates/shared/partial/head_scripts.html

```diff
@@ -7,14 +7,22 @@
 <script>
 var DJANGO_STATIC_ROOT = "{% static '' %}";
 var DJANGO_CSRF_TOKEN = $("[name=csrfmiddlewaretoken]").val();
+{% if user.is_authenticated %}
+    var DJANGO_USER_ID = "{{ user.get_username }}"
+{% else %}
+    var DJANGO_USER_ID = null;
+{% endif %}
 // Global reference to the preferences editor for non-Infusion code
 // Initialized with the results of createClusivePrefsEditor() when that's called
 var clusivePrefs;
 </script>
 
 <script src="{% static 'shared/js/lib/main.js' %}"></script>
 <script src="{% static 'shared/js/clusive-reader-prefs.js' %}"></script>
+<script src="{% static 'shared/js/clusive-prefs-django-store.js' %}"></script>
 <script src="{% static 'shared/js/clusive-prefs-panel.js' %}"></script>
+<script src="{% static 'shared/js/clusive-prefs-modal-settings.js' %}"></script>
+<script src="{% static 'shared/js/clusive-message-queue.js' %}"></script>
```

Comment:
We should perhaps update the build to bundle more of these into main.js, unless there's a reason not to.